app/controllers/DraftMetadataUploadController.scala

```diff
@@ -55,17 +55,21 @@ class DraftMetadataUploadController @Inject() (
       val token = request.asInstanceOf[Request[AnyContent]].token
       val uploadBucket = s"${applicationConfig.draft_metadata_s3_bucket_name}"
       val uploadFileName = applicationConfig.draftMetadataFileName
+      val userUploadedFile = request.body.files.headOption
       val uploadKey = s"$consignmentId/$uploadFileName"
       val noDraftMetadataFileUploaded: String = "No meta data file provided"
       val consignmentStatusInput = ConsignmentStatusInput(consignmentId, DraftMetadataType.id, Some(InProgressValue.value))
 
       def uploadDraftMetadata: IO[Result] = for {
+        _ <- IO(
+          logger.info(s"User:${token.userId} uploaded the draft metadata file '${userUploadedFile.map(_.filename).getOrElse(uploadFileName)}' for consignment:$consignmentId")
```

Comment:
I would leave out the filename from the logs in case it contains sensitive information. So change the log message to: `User ... uploaded draft metadata file for consignment ...`